Composer/ScriptHandler.php

```diff
@@ -319,7 +319,7 @@ private static function patchAcmeDemoBundleConfiguration($appDir, Filesystem $fs
 
     # with these settings you can restrict or allow access for different parts
     # of your application based on roles, ip, host or methods
-    # http://symfony.com/doc/current/book/security.html#security-book-access-control-matching-options
+    # http://symfony.com/doc/current/cookbook/security/access_control.html
```

Comment:
@xelaris Can you reopen here? The first two anchors can be fixed in the docs (see symfony/symfony-docs#4785). This one needs to be changed here since the content was moved.